src/main/java/org/janelia/saalfeldlab/n5/N5FSReader.java

```diff
@@ -136,7 +136,7 @@ private void close() throws IOException {
 	}
 
 	/**
-	 * Reads or creates the attributes map of a group or dataset.
```

Comment:
This is the place where I would prefer this to be fixed.  If there is no attributes.json file, the method should return an empty map.  That would also fix the core issue of this pull request.